mailbox/store/src/main/java/org/apache/james/mailbox/store/search/MessageSearches.java

```diff
@@ -421,26 +421,15 @@ private boolean matches(SearchQuery.HeaderCriterion criterion, MailboxMessage me
      */
     private boolean matchesAddress(SearchQuery.AddressOperator operator, String headerName,
                                    MailboxMessage message) throws MailboxException, IOException {
-        String text = operator.getAddress().toUpperCase(Locale.US);
+        String text = operator.getAddress();
```

Comment:
why?

Reply:
for example the string `Üsteliğhan` can not be well convert to Upper case with the `Locale.US` but another locale but IDK what is it. I just get the original value, pass it to the String searching method `doesMailboxContains()`. That is the place I convert it to lower case & upper case to increase the chance to match input text.

Reply:
but is it part of this commit?

Reply:
okay, do you mean put this change in another commit?

Reply:
about the `Üsteliğhan` upper case & lower case, I was wrong, just using the uppercase with Locale.US is enough. And just did comparing upper case of two strings like before.